EosGround/config/eos_schema.sql

```diff
@@ -154,4 +154,25 @@ ALTER TABLE IF EXISTS eos_schema."position"
     ON DELETE NO ACTION
     NOT VALID;
 
+
+CREATE TABLE IF NOT EXISTS eos_schema.terminal_output
+(
+    id integer GENERATED ALWAYS AS IDENTITY,
+    received_packet_id integer,
+    transmit_table_id integer UNIQUE,
```

Comment:
what is the reason for received_packet_id being not unique but transmit_table_id unique?

Reply:
This is a good question, i guess they would both be inherently unique cause they are pulling from primary keys. I will probably make received_packet_id unique as well